index.js

```diff
@@ -12,4 +12,92 @@ mobileMenu.addEventListener('click', () => {
+    linkSource: 'https//www.githubrepo',
+    linkLive: 'https//www.gitpages',
+  },
+  {
+    title: 'Keeping track of hundreds of components',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    image: '',
+    technologies: ['Ruby on Rails', 'css', 'JavaScript', 'html'],
+    linkSource: 'https//www.githubrepo',
+    linkLive: 'https//www.gitpages',
+  },
+  {
+    title: 'Project 5',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    image: '',
+    technologies: ['Ruby on Rails', 'css', 'JavaScript', 'html'],
+    linkSource: 'See Live <img src="images/Iconlive.svg"></img>',
+    linkLive: 'https//www.gitpages',
+  },
+];
+
+const workBox = document.querySelector('.card_Work_Row_1');
+const popup = document.getElementById('popup-window');
+workBox.addEventListener('click', (event) => {
+  const index = event.target.id;
+
+  const title = document.getElementById('projectTitle');
+  title.innerHTML = objWorks[index].title;
+  const description = document.getElementById('projectDescription');
+  description.innerHTML = objWorks[index].description;
+  const image = document.getElementById('projectImage');
+  image.innerHTML = objWorks[index].image;
+
+  const techs = document.getElementById('tags');
+  for (let i = 0; i < objWorks[index].technologies.length; i += 1) {
+    techs.innerHTML = `<ul><li><button>${objWorks[0].technologies[0]}</button></li></ul>
+                          <ul><li><button>${objWorks[0].technologies[1]}</button></li></ul>
+                          <ul><li><button>${objWorks[0].technologies[2]}</button></li></ul>
+                          <ul><li><button>${objWorks[0].technologies[3]}</button></li></ul>`;
+  }
+
+  popup.style.display = 'block';
+  document.body.style.overflow = 'none';
+});
+// get span element that closes popup
+const span = document.getElementById('close');
+// to close the popub
+span.onclick = function () {
+  popup.style.display = 'none';
+};
+// when click anywhere
+window.onclick = function (event) {
+  if (event.target === popup) {
+    popup.style.display = 'none';
+  }
+};
```

Comment:
- [ ] Good job on working on some suggestions from the previous reviewer. However, there some things you still need to work on before you project can be approved.
- The left alignment of the list of stack used to build the project is not in line with the design template. Kindly compare below two screenshots of your design and the figma design. :point_down: 

### Your Design(Desktop)
![your-design-desktop](https://user-images.githubusercontent.com/26597432/211160221-5f424cb4-9666-4ec5-91a6-4cd2270bd14d.png)

### figma Design(Desktop)
![figma-design](https://user-images.githubusercontent.com/26597432/211160235-51df1733-aa09-4368-9768-19bc2f19ce01.png)

### Your Design(Mobile)
![your-design-mobile](https://user-images.githubusercontent.com/26597432/211160245-7afc134f-8d1f-4cad-9421-b5cc314f6713.png)

### Figma Design(Mobile)
![figma-mobile](https://user-images.githubusercontent.com/26597432/211160262-f036f121-e1bb-46de-a051-f4d1ab01e868.png)

.  Kindly work on them and ask for another review. Best of luck.